janitor/functions/min_max_scale.py

```diff
@@ -125,24 +127,24 @@ def min_max_scale(
     if column_name is not None:
         df = df.copy()  # Avoid to change the original DataFrame.
 
-        old_feature_range = df[column_name].pipe(min_max_value, jointly)
+        old_feature_range = df[column_name].pipe(_min_max_value, jointly)
         df[column_name] = df[column_name].pipe(
-            apply_min_max,
+            _apply_min_max,
             *old_feature_range,
             *feature_range,
         )
     else:
-        old_feature_range = df.pipe(min_max_value, jointly)
+        old_feature_range = df.pipe(_min_max_value, jointly)
         df = df.pipe(
-            apply_min_max,
+            _apply_min_max,
             *old_feature_range,
             *feature_range,
         )
 
     return df
 
 
-def min_max_value(df: pd.DataFrame, jointly: bool) -> tuple:

```

Comment:
[functions/__init__.py](https://github.com/pyjanitor-devs/pyjanitor/blob/63c075ec935493faf0ca8aa19e8bd1c601258e7c/janitor/functions/__init__.py#L57) only imports `min_max_scale`.
But the docstring of inner methods would be rendered by mkdocs.
This behavior is quite different to sphinx.